src/App.jsx

```diff
@@ -14,70 +15,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState('Jam');
```

Comment:
please update `setGood` to `setSelectedGood`

Reply:
yes, thanks, I also saw that I need to do as you say in the next lesson but forget change it 😁